engo_glfw.go

```diff
@@ -317,7 +323,8 @@ func SetVSync(enabled bool) {
 	}
 }
 
-func init() {
+// Initializes application
+func InitGl() {
 	runtime.LockOSThread()
```

Comment:
`InitGl` would be called `InitGL` though, because `GL` is an acronym. Like `HTML`.